ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/frontend.hpp

```diff
@@ -40,7 +67,10 @@ namespace ngraph
 
         private:
             static std::shared_ptr<Function>
-                convert_model(const std::shared_ptr<InputModelPDPD>& model);
+                convert_each_node(const std::shared_ptr<InputModelPDPD>& model,
```

Comment:
can we completely move this function to .cpp file?

Reply:
Unfortunately, this function needs access to protected members of InputModelPDPD, so it has to be part of frontend.